CodeSparks/Areas/Identity/Pages/Account/Manage/Index.cshtml.cs

```diff
@@ -89,30 +98,18 @@ public async Task<IActionResult> OnGetAsync()
         public async Task<IActionResult> OnPostAsync()
         {
             var user = await _userManager.GetUserAsync(User);
-            if (user == null)
-            {
-                return NotFound($"Unable to load user with ID '{_userManager.GetUserId(User)}'.");
-            }
 
-            if (!ModelState.IsValid)
-            {
-                await LoadAsync(user);
-                return Page();
+            if (user == null) {
+              return NotFound($"Unable to load user with ID '{_userManager.GetUserId(User)}'.");
             }
 
-            var phoneNumber = await _userManager.GetPhoneNumberAsync(user);
-            if (Input.PhoneNumber != phoneNumber)
+            if (ModelState.IsValid)
             {
-                var setPhoneResult = await _userManager.SetPhoneNumberAsync(user, Input.PhoneNumber);
-                if (!setPhoneResult.Succeeded)
-                {
-                    StatusMessage = "Unexpected error when trying to set phone number.";
-                    return RedirectToPage();
-                }
+              _context.Update(LoggedUser);
```

Comment:
There are a couple of issues with such implementation:
1. Id is not mapped, so it will be always empty Guid (all zeros)
2. Only 2 properties are mapped, so this may delete data in the rest of the fields in database in User table.